src/tribler/gui/network/request/shutdown_request.py

```diff
@@ -0,0 +1,8 @@
+from PyQt5.QtNetwork import QNetworkRequest
+
+from tribler.gui.network.request.request import Request
+
+
+class ShutdownRequest(Request):
```

Comment:
Instead of having a separate `ShutdownRequest` class, I suggest adding a `cancellable` attribute to the base `Request` class. This makes code more generic, and allows to make `tribler.gui.network.request` a module instead of a package